web/static/httprest/connect/success.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>autokitteh HTTP/REST Integration - Success!</title>
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="/static/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="/static/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="/static/favicon-16x16.png"
+    />
+    <link rel="manifest" href="/static/site.webmanifest" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <meta name="robots" content="noindex, nofollow" />
+    <script src="success.js" defer></script>
+  </head>
+
+  <body>
+    <a href="/" target="_self">
+      <img class="banner" src="/static/banner.svg" alt="autokitteh" />
+    </a>
+
+    <table class="title">
+      <td>
+        <img class="logo" src="/static/images/http.svg" alt="Logo" />
+      </td>
+      <td><h1>HTTP / REST - Connection Created</h1></td>
+    </table>
+
+    <p class="info">Your <strong>autokitteh</strong> connection token is:</p>
+    <p><span id="token" class="copyable"></span></p>
+
+    <p class="notif" id="notif">Copied into the clipboard</p>
```

Comment:
What is `notif`?

Reply:
It's for the JS code that handles copying tokens into the clipboard (see `form.js`)